docs/source/actions.rst

```diff
@@ -298,6 +298,89 @@ SPIDERMON_SLACK_NOTIFIER_REPORT_INDEX
 .. _`Slack`: https://slack.com/
 .. _`Slack credentials`: https://api.slack.com/docs/token-types
 
+Telegram action
+============
+
+
+This action allows you to send custom messages to a `Telegram`_ channel, group or user
+using a bot when your monitor suites finish their execution.
+
+To use this action you need to:
+
+#.  Provide the `Telegram bot token`_ in your ``settings.py`` file as follows:
```

Comment:
I would avoid single-item lists, and instead join this paragraph with the one above into a single paragraph.